### PR TITLE
add "Replicate vs global services" link

### PR DESCRIPTION
**Pull request by Daxaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ This guide is intended to be a point of knowledge to everyone who wants to pass
 "docker stack deploy"
 - Manipulate a running stack of services
 - [Increase number of replicas](https://docs.docker.com/engine/reference/commandline/service_scale/)
-- Add networks, publish ports
-- Mount volumes
-- Illustrate running a replicated vs global service
+- [Add networks, publish ports](https://docs.docker.com/engine/userguide/networking/)
```
Comment (Evalle) [inline README.md]: Please, delete this link

File: README.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ This guide is intended to be a point of knowledge to everyone who wants to pass
...
+- [Mount volumes](https://docs.docker.com/engine/admin/volumes/volumes/)
```
Comment (Evalle) [inline README.md]: And this one as well :)

**Comment by Daxaker** [comment]
Done.

**Comment by Evalle** [comment]
LGTM, thanks for the contributing!

**Pull request merged**